recipes/cdecimal/meta.yaml

```diff
@@ -0,0 +1,46 @@
+{% set name = "cdecimal" %}
+{% set version = "2.3" %}
+{% set sha256 = "d737cbe43ed1f6ad9874fb86c3db1e9bbe20c0c750868fde5be3f379ade83d8b" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: http://www.bytereef.org/software/mpdecimal/releases/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  skip: True  # [not py2k]
+  script: python setup.py install
+
+requirements:
+  build:
+    - python
+    - toolchain
+
+  run:
+    - python
+
+test:
+  imports:
+    - cdecimal
```

Comment:
Consider adding the [`conda-inspect`](https://github.com/conda-forge/nco-feedstock/blob/master/recipe/meta.yaml#L47-L48) tests b/c here.